model/version.go

```diff
@@ -356,6 +371,42 @@ func VersionGetHistory(versionId string, N int) ([]Version, error) {
 	return versions, nil
 }
 
+type MainlineCommitVersionOptions struct {
+	Limit           int  `json:"limit"`
+	SkipOrderNumber int  `json:"skipOrderNumber"`
+	Activated       bool `json:"activated"`
+}
+
+func GetMainlineCommitVersionsWithOptions(projectName string, opts MainlineCommitVersionOptions) ([]Version, error) {
```

Comment:
Can we write a unit test for this? I'd like to know if you have a Version inserted with Activated not set, if it gets included in the Activated: false match; otherwise if Activated is false we likely want to query both "false and unset"